### PR TITLE
Add post-install hook for rake db:migrate

### PR DESCRIPTION
**Pull request by fabubaker**
Additionally, a few miscellaneous fixes are bundled with this PR:

- Fixes .gitignore to only ignore .yaml files in helm/ directory
- Fixes typo in rayyan-store-proxy helm test

### CODE REVIEW

File: helm/rayyan/templates/jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{{- range $name, $hook := $root.Values.hooks }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ $name }}
+  labels:
+    app.kubernetes.io/name: {{ $name }}
+  annotations:
+    helm.sh/hook: {{ $hook.types }}
+    helm.sh/hook-weight: "{{ $hook.weight }}" # weight must be a string
```
Comment (hammady) [inline helm/rayyan/templates/jobs.yaml]: Can this have a safe default? Or the null is an accepted default?

Comment (fabubaker) [reply helm/rayyan/templates/jobs.yaml]: A safe default is `"0"`. I will add it.

File: helm/rayyan/templates/jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{{- range $name, $hook := $root.Values.hooks }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ $name }}
+  labels:
+    app.kubernetes.io/name: {{ $name }}
+  annotations:
+    helm.sh/hook: {{ $hook.types }}
+    helm.sh/hook-weight: "{{ $hook.weight | default 0 }}" # weight must be a string
```
Comment (hammady) [inline helm/rayyan/templates/jobs.yaml]: The default is not a string though.

Comment (fabubaker) [reply helm/rayyan/templates/jobs.yaml]: The surrounding quotes ensure that whatever value is passed in becomes a string.

**Pull request merged**